policies.go

```diff
@@ -948,19 +948,37 @@ func (host selectedHostPoolHost) Mark(err error) {
 }
 
 type dcAwareRR struct {
-	local           string
-	localHosts      cowHostList
-	remoteHosts     cowHostList
-	lastUsedHostIdx uint64
+	local            string
+	localHosts       cowHostList
+	remoteHosts      cowHostList
+	lastUsedHostIdx  uint64
+	permitDCFailover bool
+}
+
+type dcFailoverEnabledPolicy interface {
+	setDCFailoverEnabled()
+}
+
+type dcAwarePolicyOption func(p dcFailoverEnabledPolicy)
+
+func HostPolicyOptionEnableDCFailover(p dcFailoverEnabledPolicy) {
+	p.setDCFailoverEnabled()
```

Comment:
Let's change meaning of it on opposite (and name as well) to do not change default behavior.

Reply:
I already pushed code with the changes

Reply:
thanks, but underlying variable did not change it's name and also let's make it's value as `false` by default and to be changed to `true` via this option